config/builder.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: builders.build.pivotal.io
+  name: builders.kpack.io
 spec:
-  group: build.pivotal.io
+  group: kpack.io
   version: v1alpha1
   names:
     kind: Builder
     singular: builder
     plural: builders
     shortNames:
-    - cnbbuilder
-    - cnbbuilders
     - bldr
-    - bldrs
+    - custombuilder
```

Comment:
These seem unnecessary. While we have to opportunity can we just get rid of the short names?

Reply:
Hmm. I left those in so if users have the habit of typing `kubectl get custombuilder` it still works. However, maybe we should just rip the band-aide off.

Reply:
I'm in favor of ripping off the band-aid. This change is already breaking, I figured it would be a good time to pile on some more breaking changes.

Reply:
Cool cool. I think I agree. Will update.